### DISCUSSION TITLE
Position of blinds indicator

### DISCUSSION THREAD

**Discussion by jbmk2** [home-assistant/frontend#19194] [Configuration]
Some manufacturers consider 100% open and 0% closed, but others consider the opposite 0% oppen, 100% closed.
It was interesting to have the possibility to adjust that (invert) in pannel

**Comment by lencioni** [comment]
I have some SmartWings top-down bottom-up blinds that I have added to Home Assistant via Matter, and it would be nice to invert the top so it reports as closed when it is all the way up.
